### DISCUSSION TITLE
More accurate system requirements

### DISCUSSION THREAD

**Discussion by time-bomb-timmy** [KelvinTegelaar/CIPP#1312] [Q&A] [answered]
For those of us NOT familiar with the vague requirements shown in the documentation I will ask.
**GitHub**
Will a single free GitHub account provide enough storage for both of the forked repos? If not, is a single 'Team' GitHub account sufficient?
Do the forked repos have to be 'public'?

**Microsoft Azure**
Will a 'free' Azure 'subscription' work? My guess is no as all of the things mentioned in the manual install guide (Static Web Apps, Functions, Key Vault) have varying degrees of pricing.
What is a recommended Azure 'subscription' for an MSP with 50-100 clients?
I'm sure I'm approaching 'RTFM' zone, but Microsofts licensing and Azure explanations are confusing enough.

Bonus questions:
**CIPP**
How does one become a sponsor?
Are there ways to have you host this other than sponsorship?

If you want more people to implement this, make it a little more accessible with some recommendations.
Thanks in advance.
Tim

**Comment by KelvinTegelaar** [comment, marked as answer]
There's nothing vague about the system requirements, if you don't understand it might be best to just read up on the utilized technologies. With current about 3k users I think we're good though. 👍
